### PR TITLE
SwapneelM-patch-2-djsce

### PR DESCRIPTION
**Pull request by SwapneelM**
Added djsce.txt for the new domain of Dwarkadas J. Sanghvi College of Engineering 'www.djsce.edu.in' where students have started getting allotted official University email addresses.

### CODE REVIEW

**Comment by philipto** [comment]
@SwapneelM The website at http://www.djsce.edu.in says "This Website is Under Construction."
Please provide us with an official address of the university website, which has to work.

**Comment by philipto** [comment]
@SwapneelM I am sorry, this request remains unanswered for a while. I close it with no merging. Please reopen if you have an update which I have requested for in a comment above. Thank you for your understanding!

**Comment by SwapneelM** [comment]
@philipto 
**tl;dr**
All I can ask is for you to look up both domains "www.djsce.ac.in" and "www.djsce.edu.in" on whois domain lookup. You will find both registered with the same details, to the same admin email address belonging to Shri Vile Parle Kelavani Mandal (S.V.K.M.), the trust which runs our institution. I have verified the same already.


Sir, the website is under construction, however they have provided us with student email addresses at the same domain name. We haven't been informed as to exactly when the website will go live. Only Professors are provided email addresses at the domain "www.djsce.ac.in" while student emails are available only at "www.djsce.edu.in"
We have requested multiple times to at least involve students in developing the website so it can be done faster and we have an idea of what's going on however we received no response as to who is in charge of developing the website. 
All I can ask is for you to look up both domains on whois. You will find both registered with the same details, to the same admin email address which is Shri Vile Parle Kelavani Mandal, the trust which runs the institution. I have verified the same. 
I would urge you to reopen and merge the pull request as there are a lot of us students with email addresses at the same domain wishing to avail of the JetBrains software but cannot do so.

**Comment by philipto** [comment]
@SwapneelM Thank you for the details! You are right, both domains are registered for SHRI VILE PARLE KELAVANI MANDAL'S DWARKADAS J. SANGHVI COLLEGE OF ENGINEERING.
I found the university official website at **www.djsce.ac.in**, and the contact emails on the website are all in **djsce.ac.in** domain. However, there are some evidence in the internet (e.g. https://sites.google.com/a/djsce.edu.in/djgoogle/faq) that the university provides the students with a Google ID in **djsce.edu.in** domain. Thus, I reopen and merge the request. 

The problem with initial request was due to you have mentioned *new domain of Dwarkadas J. Sanghvi College of Engineering 'www.djsce.edu.in'*, and the website on this address looked suspicious.
